Automattic/jetpack
# Boost: Update Image Guide tracking to use Pixel instead of ajax requests

Fixes https://github.com/Automattic/boost-cloud/issues/234

## Proposed changes:

* Update tracking UI interactions in the front-end for Image Guide, to happen via Pixel tracking, instead of sending requests to the server.

### Other information:


## Jetpack product discussion



https://github.com/Automattic/boost-cloud/issues/234

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup this version on Boost on a website (you need a valid Jetpack connection);
* Enable Image Guide;
* While logged-in, view a page with images in the front-end;
* Check the network tab while the page is loading - you should see requests to `http://pixel.wp.com/t.gif`.
